sessionctx/variable/varsutil.go

```diff
@@ -400,6 +407,27 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 			return "", ErrUnsupportedValueForVar.GenWithStackByArgs(name, value)
 		}
 		return upVal, nil
+	case TiDBInitChunkSize:
+		v, err := strconv.Atoi(value)
+		if err != nil {
+			return value, ErrWrongTypeForVar.GenWithStackByArgs(name)
+		}
+		if v <= 0 {
```

Comment:
Can we put the check of `err != nil ` in this branch?

Reply:
no, it's use different message.